UIKitDZ/UIKitDZ/Model.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Model.swift
+//  UIKitDZ
+//
+//  Created by Владислав Лымарь on 22.09.2022.
+//
+
+import Foundation
+import UIKit
+///
+struct Model {
+    var booking: Bool?
```

Comment:
```suggestion
    var isBooking: Bool?
```
приставка is у всех болов